SmartDeviceLink/SDLRadioControlCapabilities.h

```diff
@@ -12,6 +12,7 @@ NS_ASSUME_NONNULL_BEGIN
 @interface SDLRadioControlCapabilities : SDLRPCStruct
 
 - (instancetype)initWithModuleName:(NSString *)moduleName radioEnableAvailable:(BOOL)radioEnableAvailable radioBandAvailable:(BOOL)radioBandAvailable radioFrequencyAvailable:(BOOL)radioFrequencyAvailable hdChannelAvailable:(BOOL)hdChannelAvailable rdsDataAvailable:(BOOL)rdsDataAvailable availableHDsAvailable:(BOOL)availableHDsAvailable stateAvailable:(BOOL)stateAvailable signalStrengthAvailable:(BOOL)signalStrengthAvailable signalChangeThresholdAvailable:(BOOL)signalChangeThresholdAvailable;
```

Comment:
This should be deprecated using `deprecated_msg((""))` similar to other places in the project